.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
The .dockerignore is like the .gitignore and allows to build Docker images much faster.